jormungandr/Cargo.toml

```diff
@@ -63,6 +63,7 @@ jormungandr-utils = { path = "../jormungandr-utils" }
 strfmt = "0.1"
 gtmpl = "0.5.6"
 mktemp = "0.4.0"
+slog-gelf = { git = "https://github.com/jbgi/slog-gelf", rev = "18197f7d5c9f439197111299509f02b24efe8985" }
```

Comment:
unless you're planning to maintain it yourself going forward, it probably make sense  copied in this repository

Reply:
I think it would be better to make a fork in our GitHub org. But that's only second best to sending patches upstream, in case the original maintainer no longer works on the project.

Reply:
https://github.com/lancastr/slog-gelf/issues/1#issuecomment-503046771

Reply:
Ask and ye shall receive :slightly_smiling_face: